### PR TITLE
Updated doc for date filter.

### PR DESCRIPTION
**Pull request by theofidry**
Doc for PR #86.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +55,10 @@ This bundle is documented and tested with Behat (take a look at [the `features/`
   2. [Returning a paged collection](Resources/doc/data-providers.md#returning-a-paged-collection)
   3. [Supporting filters](Resources/doc/data-providers.md#Supporting-filters)
 5. [Filters](Resources/doc/filters.md)
-  1. [Adding Doctrine ORM filters](Resources/doc/filters.md#adding-doctrine-orm-filters)
-  2. [Creating custom filters](Resources/doc/filters.md#creating-custom-filters)
+  1. [Adding Doctrine ORM filters](Resources/doc/filters.md#doctrine-orm-search-filter)
+  2. [Adding Doctrine ORM filters](Resources/doc/filters.md#doctrine-orm-date-filter)
```
Comment (dunglas) [inline README.md]: Wrong titles.

**Comment by theofidry** [comment]
Fixed.

**Comment by dunglas** [comment]
I think a structure like the following has more sense:

# Doctrine ORM
## Search filter
## Date filter
## Order filter

**Comment by theofidry** [comment]
done

**Comment by dunglas** [comment]
Thank you @theofidry

**Comment by theofidry** [comment]
Always a pleasure

**Pull request merged**